README.md

```diff
@@ -1,2 +1,11 @@
 # git_training
-Git training from the NHS-R community
+Git training from the NHS-R community training steps:
+
+1. Download [R](https://cran.r-project.org/bin/windows/base/) and [RStudio desktop](https://www.rstudio.com/products/rstudio/download/) or create an account on [RStudio cloud](https://login.rstudio.cloud/login?redirect=%2F).
+2. Install Git via this [linnk](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) if you are using RStudio desktop.
+3. Use this repository or create a [new repository.](./guides/how_to_create_a_repo.md) 
+4. Follow the introduction to Git and version control for R users [guide](./guides/introduction_to_git.md). Alternatively, for Python users see [here.](https://github.com/NHSDigital/rap-community-of-practice/blob/main/development-approach/01_intro-to-git.md).
+5. Use the [Let's Git started guide](https://happygitwithr.com/index.html) for further info on connecting to Git via RStudio, troubleshooting tips and Git/R features.
+
+## Practice 
```

Comment:
Maybe we could rename this section 'What's next?', and include the line above, and also this bit about practice.